### PR TITLE
XML Codestyle

### LINKED ISSUE #229: Admin menu presets
If you look at the xml files for the menu presets they are a mess of spaces and tabs and different indentations. 

image

We have a codestyle for the xml files and these must be corrected

### PR DESCRIPTION
**Pull request by Hackwar**
This consistently uses spaces for indentation for all menu presets instead of tabs. Fixes #229

### CODE REVIEW

**Review by chmst — DISMISSED** [review]

**Comment by brianteeman** [comment]
Sometimes the last line of a menuitem ends with a >

https://github.com/joomla/backend-template/blob/1b138dc0d3a5e27c8c38b565f0ebd0a372bf52b9/administrator/components/com_menus/presets/joomla.xml#L50

And sometimes it doesn't end with a >

https://github.com/joomla/backend-template/blob/1b138dc0d3a5e27c8c38b565f0ebd0a372bf52b9/administrator/components/com_menus/presets/joomla.xml#L59


Is that really correct?

**Comment by brianteeman** [comment]
Pretty sure that the code style is for us to use tabs not spaces

**Comment by Hackwar** [comment]
We don't have an official, enforced codestyle for XML, but you are right, that the original files were with tabs and not spaces. I will correct that soon.

**Comment by brianteeman** [comment]
All the xml files are with tabs - I know because at some point in time I have updated 99% of them

**Comment by brianteeman** [comment]
Thanks for changing to tabs. What about my other question https://github.com/joomla/backend-template/pull/265#issuecomment-496303444

**Comment by fancyFranci** [comment]
> Sometimes the last line of a menuitem ends with a >
> 
> https://github.com/joomla/backend-template/blob/1b138dc0d3a5e27c8c38b565f0ebd0a372bf52b9/administrator/components/com_menus/presets/joomla.xml#L50
> 
> And sometimes it doesn't end with a >
> 
> https://github.com/joomla/backend-template/blob/1b138dc0d3a5e27c8c38b565f0ebd0a372bf52b9/administrator/components/com_menus/presets/joomla.xml#L59
> 
> Is that really correct?

Yes this should be consistent. I would prefer the '>' on a separate line, but which style should we use in Joomla?

**Comment by brianteeman** [comment]
See https://developer.joomla.org/coding-standards/xml.html for the documented xml codestyle

**Comment by fancyFranci** [comment]
Thank you for the link! Then we should put it on the next line.

**Comment by Hackwar** [comment]
@brianteeman I've changed the codestyle to the style you linked above. Can you have a look if you are happy with the results?

**Review by bembelimen — DISMISSED** [review]

File: administrator/components/com_menus/presets/menus.xml [hunk 1]
```diff
@@ -1,71 +1,76 @@
...
+		sql_target="self"
+		>
+		<menuitem
+			title="{sql:title} "
+			type="component"
+			element="com_menus"
+			link="index.php?option=com_menus&amp;view=items&amp;menutype={sql:menutype}"
+			icon="{sql:icon}"
+			class="class:menu"
+			quicktask="index.php?option=com_menus&amp;task=item.add&amp;menutype={sql:menutype}">
```
Comment (brianteeman) [inline administrator/components/com_menus/presets/menus.xml]: just this one needs fixing

Comment (Hackwar) [reply administrator/components/com_menus/presets/menus.xml]: Thanks, done.

**Comment by brianteeman** [comment]
codestyle is all good now

**Review by bembelimen — APPROVED** [review]

**Pull request merged**